containers/base/Dockerfile

```diff
@@ -15,98 +15,97 @@
 FROM debian:jessie
 MAINTAINER Google Cloud DataLab
 
+# Directory "py" may be empty and in that case it will git clone pydatalab from repo
+ADD pydatalab /datalab/lib/pydatalab
+ADD nbconvert /datalab/nbconvert
+
+# Container configuration
+EXPOSE 8080
+
+# Path configuration
+ENV PATH $PATH:/tools/node/bin:/tools/google-cloud-sdk/bin
+ENV PYTHONPATH /env/python
+
 # Setup OS and core packages
 RUN echo "deb-src http://ftp.us.debian.org/debian testing main" >> /etc/apt/sources.list && \
     apt-get update -y && \
-    apt-get install --no-install-recommends -y -q \
-        curl wget unzip git vim build-essential ca-certificates pkg-config \
-        libatlas-base-dev liblapack-dev gfortran \
-        libpng-dev libfreetype6-dev libxft-dev \
-        rsync \
-        libxml2-dev \
-        python2.7 python-dev python-setuptools python-zmq openssh-client && \
+    apt-get install --no-install-recommends -y -q python unzip ca-certificates build-essential \
+    libatlas-base-dev liblapack-dev gfortran libpng-dev libfreetype6-dev libxft-dev libxml2-dev \
+    python2.7 python-dev python-setuptools openssh-client wget git pkg-config && \
     easy_install pip && \
-    mkdir -p /tools
+    mkdir -p /tools && \
 
 # Save GPL source packages
-RUN mkdir -p /srcs && \
+    mkdir -p /srcs && \
     cd /srcs && \
     apt-get source -d python-zmq wget git ca-certificates pkg-config libpng-dev && \
```

Comment:
What is the reason for removing the "python-zmq" dependency but keeping its source code?

In particular, do we still have "python-zmq" being installed in the image as a transitive dependency of something else we are installing? If so, then I would rather keep that explicit rather than implicit.